terraform/email.tf

```diff
@@ -19,12 +19,12 @@ resource "aws_sesv2_configuration_set" "default" {
 }
 
 data "aws_sns_topic" "datadog_forwarder" {
-  count = var.ses_datadog_events_enabled ? 1 : 0
+  count = var.email_enable_tracking && var.ses_datadog_events_enabled ? 1 : 0
```

Comment:
I suppose I could have set `ses_datadog_events_enabled` to false instead of introducing a new variable but it felt like that variable was not meant to be used as a feature flag.